base/ubi9/Dockerfile

```diff
@@ -19,17 +19,26 @@ LABEL io.openshift.expose-services=""
 
 USER 0
 
+ENV HOME=/home/tooling
+RUN mkdir -p /home/tooling/
+
 # Removed because of vulnerabilities: git-lfs
-RUN dnf install -y diffutils git iproute jq less lsof man nano procps \
-    perl-Digest-SHA net-tools openssh-clients rsync socat sudo time vim wget zip && \
+RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    dnf install -y diffutils git git-lfs iproute jq less lsof man nano procps p7zip p7zip-plugins \
+        perl-Digest-SHA net-tools openssh-clients rsync socat sudo time vim wget zip stow && \
     dnf update -y && \
     dnf clean all
 
+## podman buildah skopeo
+RUN dnf -y reinstall shadow-utils && \
+    dnf -y install podman buildah skopeo fuse-overlayfs && \
+    dnf clean all
+
 ## gh-cli
 RUN \
     TEMP_DIR="$(mktemp -d)"; \
     cd "${TEMP_DIR}"; \
-    GH_VERSION="2.23.0"; \
+    GH_VERSION="2.45.0"; \
```

Comment:
This was done to match the GH version in the ubi8 image: https://github.com/devfile/developer-images/blob/0eb879a7fccd12da128f89b99a986292a0b9091a/base/ubi8/Dockerfile#L42